process/archiving.md

```diff
@@ -0,0 +1,30 @@
+# CNCF Project Archiving Process v1.0
+
+Open source projects have a lifecycle and there are times that projects become inactive due to a variety of reasons. There are also cases where a project may no longer want to be supported by the TOC, or the TOC may no longer wish to recommend the use of a project.
+
+## Archiving Criteria
+
+When voting on a proposal to archive a project, TOC members may wish to consider whether the project continues to meet the criteria for CNCF acceptance. The TOC may also look at activity levels in the project, although it is important to note that there is a difference between a mature project that doesn't get much attention any more but is stable, versus a project that is inactive.
+
+## Voting Process
+
+To archive a project:
+
+* A proposal must be put forth to the TOC repo
+* The TOC will inform the project maintainers, CNCF end user community and wider community of all archiving proposals
+* The proposal must remain open for at least 2 weeks of discussion after the maintainers are informed.
+* A vote must be finalized with 2/3 approval from the TOC
+
+## Archiving Process
+
+What does archiving for a CNCF project mean?
+
+* CNCF will no longer provide support for the project via service desk
+* CNCF will list archived projects online
+* Trademarks and domain names of archived projects are still hosted by the CNCF and the Linux Foundation
+* CNCF can provide services such as documentation updates to help transition users.
+* Other CNCF marketing activities will no longer be provided for the project
+
+## Reactivating an Archived Project
+
+Any project can be reactivated into CNCF by following the normal project [proposal](https://github.com/cncf/toc/blob/master/process/project_proposals.adoc) and [sandbox](https://github.com/cncf/toc/blob/master/process/sandbox.md) process.
```

Comment:
If you re-activate a project, are the options either incubating or sandbox or just sandbox?

Reply:
I think either is possible - we would assess it like a new project coming in. (I suppose in theory projects could even come in as graduated...)

Reply:
imho either should be possible

Reply:
addressing this in an updated commit